game/templates/game/board.html

```
@@ -167,6 +175,9 @@
         </div>
         
         <div class="auth-buttons">
+            <button id="themeToggleBtn" class="theme-toggle-btn" title="Toggle Day/Night" aria-label="Toggle Day/Night mode">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)